### ISSUE TITLE
More detailed info in readme?

### ISSUE THREAD

**Issue by impressivewebs** [bcinarli/caffeine#6]
It would be nice if the README file on the main repo page showed which mixins/functions are included in this. Otherwise, there isn't much incentive for anyone to investigate this further or try this out. Even just a simple list of the mixin names might be enough, but probably good to list each one with a description and example of use. Thanks!

**Comment by bcinarli** [comment]
Absolutely @impressivewebs, up to now, I was adding detailed inline documentation and inline notes and a sample [html](https://github.com/bcinarli/caffeine/blob/master/tests/index.html) in tests. I'm working on the visible documentation and mixin/function list most likely, it will finish sometime in next week.

**Comment by impressivewebs** [comment]
Great. I was hoping to include it in [my newsletter](http://webtoolsweekly.com/) this week, but probably best to wait until you have a better description in the readme. Thanks!

**Comment by bcinarli** [comment]
I'd be very appreciated. Also did you checked the architectural approach in the [Melange](https://github.com/bcinarli/melange) and its [sample](https://github.com/bcinarli/melange-sample). I'm also writing their documentations :)

**Comment by impressivewebs** [comment]
No, I don't think I've seen Melange. Thanks, I have saved it for inclusion. :)
